com.github.jeromerobert.pdfarranger.yaml

```diff
@@ -0,0 +1,66 @@
+---
+id: com.github.jeromerobert.pdfarranger
+runtime: org.gnome.Platform
+runtime-version: '3.36'
+sdk: org.gnome.Sdk
+command: pdfarranger
+# These renames will be obsolete with 
+# https://github.com/jeromerobert/pdfarranger/issues/158
+# Currently known bug: Missing icon in about dialog (Will be fixed by #158.)
+rename-icon: pdfarranger
+rename-desktop-file: pdfarranger.desktop
+rename-appdata-file: pdfarranger.appdata.xml
+finish-args:
+- --socket=wayland
+- --socket=x11
+- --share=ipc
+- --filesystem=home
+modules:
+- shared-modules/intltool/intltool-0.51.json
+- shared-modules/openjpeg/openjpeg.json # dependency of poppler
+- name: poppler
+  buildsystem: cmake-ninja
+  sources:
+  - url: https://poppler.freedesktop.org/poppler-0.86.1.tar.xz
+    sha256: af630a277c8e194c31339c5446241834aed6ed3d4b4dc7080311e51c66257f6c
+    type: archive
+  config-opts:
+  - -DENABLE_UTILS=OFF
+  - -DENABLE_CPP=OFF
+  - -DENABLE_QT5=OFF
+- name: python-distutils-extra
+  buildsystem: simple
+  sources:
+  - url: https://launchpad.net/python-distutils-extra/trunk/2.39/+download/python-distutils-extra-2.39.tar.gz
+    sha256: 723f24f4d65fc8d99b33a002fbbb3771d4cc9d664c97085bf37f3997ae8063af
+    type: archive
+  build-commands:
+  - pip3 install --prefix=/app .
+- name: pikepdf
+  buildsystem: simple
+  build-commands:
+  - pip3 install --no-index --prefix=$FLATPAK_DEST lxml-*.whl pikepdf-*.whl
+  sources:
+  - type: file
+    url: https://files.pythonhosted.org/packages/38/e8/9cce836a4b910e3928125d5566592b6531a791cfd7e90636d5104c497c65/pikepdf-1.10.3-cp37-cp37m-manylinux2010_x86_64.whl
```

Comment:
The whl files are x86_64 only. This might be helpful if you want to distribute it for x86_64 only https://github.com/flathub/flathub/wiki/App-Maintenance#flathubjson

Reply:
Alternatively use pip generator: https://github.com/flatpak/flatpak-builder-tools/tree/master/pip

Reply:
I tried a lot to make the pip-generator work with pikepdf, first before starting this pull request and then again yesterday, but I just couldn't solve this. I'm quite sure its the same issue as https://github.com/flatpak/flatpak-builder-tools/issues/90 and https://github.com/flatpak/flatpak-builder-tools/issues/100 so I'll wait and see how these get resolved and build x86_64 only in the meantime if that's okay with you. (I lost my patience with the pip-generator tbh.)

Reply:
Why not using the tarballs of pikepdf from https://github.com/pikepdf/pikepdf/releases which shouldn't be x86_64 only